docs/howto/openstack/nova/server-group.md

```diff
@@ -68,9 +72,11 @@ $ openstack server show -c fault -c status <server_id>
 +--------+--------------------------------------------------+
 ```
 
-This is normal as {{brand}} cannot schedule the server on a different physical compute node because there already is a server in the server group on every node. The same scheduling error and "fault" message will occur when using a server group with a policy of `affinity` as well, when you create more servers than a physical compute node can host.
+This is normal as {{brand}} cannot schedule the server on a different physical compute node because there already is a server in the server group on every node.
```

Comment:
Sounds a bit better inside muh head.

```suggestion
This is normal as {{brand}} cannot schedule the server on a different physical compute node because there is already a server in the server group on every node.
```